src/main/resources/assets/commandblockide/lang/zh_cn.json

```diff
@@ -1,23 +1,23 @@
 {
     "arguments.editfunction.tag.unsupported": "无法编辑函数标签",
     "commands.editfunction.failed.modNotInstalled": "您必须安装 Command Block IDE 模组",
-    "commandBlockIDE.apply": "运用",
-    "commandBlockIDE.type.redstone": "普通",
-    "commandBlockIDE.type.redstoneConditional": "普通, 条件限制",
+    "commandBlockIDE.apply": "应用",
+    "commandBlockIDE.type.redstone": "脉冲",
+    "commandBlockIDE.type.redstoneConditional": "脉冲, 条件制约",
     "commandBlockIDE.type.auto": "循环",
-    "commandBlockIDE.type.autoConditional": "循环, 条件限制",
+    "commandBlockIDE.type.autoConditional": "循环, 条件制约",
     "commandBlockIDE.type.sequence": "连锁",
-    "commandBlockIDE.type.sequenceConditional": "连锁, 重复",
+    "commandBlockIDE.type.sequenceConditional": "连锁, 条件制约",
     "commandBlockIDE.unloaded": "未加载",
     "commandBlockIDE.lastOutput.on": "跟踪输出",
     "commandBlockIDE.lastOutput.off": "不跟踪输出",
     "commandBlockIDE.lastOutput.none": "无输出",
     "commandBlockIDE.narrator.editorIndex": "数字 %d",
-    "commandBlockIDE.saveFunction.success.file": "保存函数 '%s' 到文件。",
-    "commandBlockIDE.saveFunction.failed.resourceManager": "在修改 '%s' 时发生错误: 资源管理器的类型不正确。",
-    "commandBlockIDE.saveFunction.failed.missingNamespace": "在修改 '%s' 时发生错误: 资源管理器中没有 '%s' 命名空间。",
-    "commandBlockIDE.saveFunction.failed.noResourcePack": "在修改 '%s' 时发生错误: 在任何数据包中无法找到此函数。",
-    "commandBlockIDE.saveFunction.failed.ioException": "在修改 '%s' 时发生错误: 发生了 IO 错误。",
-    "commandBlockIDE.saveFunction.failed.zipNotSupported": "在修改 '%s' 时发生错误: 尚不支持 .zip 文件中的数据包。 ",
-    "commandBlockIDE.saveFunction.failed.packClassNotSupported": "在修改 '%s' 时发生错误: 不支持资源包类 '%s'。"
+    "commandBlockIDE.saveFunction.success.file": "函数 '%s' 保存成功。",
+    "commandBlockIDE.saveFunction.failed.resourceManager": "保存 '%s' 时发生错误: 数据包类型错误。",
```

Comment:
When using Google Translate, this translates back to English as:

> Error saving '%s': wrong packet type.

This does not match the original, which says “Resource manager was not the correct type.”

The older text translates back to “An error occurred while modifying '%s': The resource manager is of an incorrect type.” It would be great if these two could be merged, maybe something like this:

> 保存 '%s' 时发生错误: 资源管理器的类型不正确。

The above translates back to English as:

> Error saving '%s': Incorrect type of resource manager.

This seems closer to the original to me, but I don't know Chinese, so please let me know whether or not this makes sense.

Reply:
Does `Resource manager` mean `A class in Java` or `Manager like Explorer.exe in Windows`?
Thanks :)

Reply:
It's a class in Java.

Reply:
兄啊，最后一行多加了个, 导致没法用（

Reply:
> 兄啊，最后一行多加了个, 导致没法用（

https://github.com/arm32x/command-block-ide/pull/13
这个是最近的 Translation Pull request

Reply:
可是即使放到Release里面也没去掉那个,
也意味着即便是简中也不会调用zh_cn.json

Reply:
> > 兄啊，最后一行多加了个, 导致没法用（
> 
> #13 这个是最近的 Translation Pull request

可是即使放到Release里面也没去掉那个,
也意味着即便是简中也不会调用zh_cn.json